share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -237,7 +244,7 @@ export def mkRemoteCacheRunner (rscApi: RemoteCacheApi) (hashFn: RunnerInput =>
         #     by the same job. If so, upload the target as a "hidden" output file. On rehydration
         #     retore the file as normal but don't list it in the outputs.
         ## -------------------------------------------------------------------------------------- ##
-        Pass (RunnerOutput inputs outputs Nil predict)
+        Pass (RunnerOutput inputs outputs cleanableOutputs predict)
```

Comment:
Ah this Nil should have triggered something in our brains during code review I guess.

Reply:
Yeah, I had the same thought. It was subtle but it should have been a flag